src/components/layout/NavBar.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+ <div>
+  <b-navbar toggleable="lg" type="dark" variant="info">
+    <b-navbar-brand exact to="/" >
+    <img class="logo" src="@/assets/freqtrade-logo.png" alt="Home Logo" />
+          <span class="navbar-brand-title">Freqtrade UI</span>
+    </b-navbar-brand>
+
+    <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
+
+    <b-collapse id="nav-collapse" is-nav>
+      <b-navbar-nav>
+        <b-nav-item to="/trade">Trade</b-nav-item>
+        <b-nav-item to="/about">About</b-nav-item>
+      </b-navbar-nav>
+
+      <!-- Right aligned nav items -->
+      <b-navbar-nav class="ml-auto">      
+          <li class="nav-item" v-if="loggedIn">
+            <b-nav-item-dropdown right>
+                <template v-slot:button-content>
+                  <b-avatar button @click="onClick" src="https://placekitten.com/300/300"></b-avatar>
```

Comment:
I will probably replace this "kitten" logo with the freqtrade logo (or a "hamburger" menu) ...

Using a "real" avatar here seems odd as this is not your typical website - but it's a monitoring dashboard for the user alone (and maybe the 2 people who he shares it with) ...

Reply:
Hehe. Yea, you're right, that was more of a placeholder.